protoc_docs/bin/py_docstring.py

```diff
@@ -50,7 +50,7 @@ def main(input_file=sys.stdin, output_file=sys.stdout):
 
                 # Seriously, protoc, an insertion point but no trailing
                 # comma before it?
-                content=',\n__doc__ = """{docstring}""",'.format(
+                content=',\n\'__doc__\' : """{docstring}""",'.format(
```

Comment:
Is this due to a protoc change?

Reply:
I believe so. I was using libprotoc 3.6.1 , ~7months old, when I regenerated the IAM pb files and using this plugin. They couldn't be interpreted  because of syntax errors induced by this generated section.

Reply:
I just regen'd with libprotoc 3.9.0 and does the same thing, uses object notation rather than a dict()

Reply:
Verified on my end.